docs/data-sources/remote_network.md

```diff
@@ -0,0 +1,32 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_remote_network Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  A Remote Network represents a single private network in Twingate that can have one or more Connectors and Resources assigned to it. You must create a Remote Network before creating Resources and Connectors that belong to it. For more information, see Twingate's documentation https://docs.twingate.com/docs/remote-networks.
+---
+
+# twingate_remote_network (Data Source)
+
+A Remote Network represents a single private network in Twingate that can have one or more Connectors and Resources assigned to it. You must create a Remote Network before creating Resources and Connectors that belong to it. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/remote-networks).
+
+## Example Usage
+
+```terraform
+data "twingate_remote_network" "foo" {
+  id = "<your network's id>"
```

Comment:
We should change this to the following to make it more explicit:

`id = "<remote network id>"`

And then we should clarify where this ID comes from, which I'll add in a separate comment.